blueprints/ember-cli-blueprint-test-helpers/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
   },
 
   addMochaToPackage() {
-    let pkg = fs.readJsonSync(path.join(__dirname, '../../package.json'));
+    let pkg = require('../../package.json');
 
     return this.addPackageToProject('mocha', pkg.devDependencies['mocha']);
   },
```

Comment:
@kellyselden I'm not sure this is a good idea, because require has caching builtin...

Reply:
wow good catch. reverting.